x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/authz/AuthorizationServiceTests.java

```diff
@@ -947,7 +947,7 @@ public void testUnknownRoleCausesDenial() {
                         + action
                         + "] is unauthorized"
                         + " for user [test user]"
-                        + " with effective roles [] (assigned roles [non-existent-role] were not found),"
+                        + " with effective roles [] (assigned roles [non-existent-role] were not found)"
```

Comment:
Just curious: Is the removal of the comma intentional?

Reply:
For reference, we synced on this topic too.
The root cause is that the refactor changed the access denied message to show [*,-*] as denied indices. We remediated that in [f3cfbe4](https://github.com/elastic/elasticsearch/pull/90141/commits/f3cfbe42d1589fc971736b788c5a7089ae1d6f47) [aae9fa5](https://github.com/elastic/elasticsearch/pull/90141/commits/aae9fa5f994745beaa951f89c820cb9241c43118) and [62d5c1e](https://github.com/elastic/elasticsearch/pull/90141/commits/62d5c1e30a42dd14db398f27487419f9e9364fc8) .